Game/Source/GDKTestGymsFunctionalTests/GDKTestGymsFunctionalTests.cpp

```diff
@@ -2,8 +2,20 @@
 
 #include "GDKTestGymsFunctionalTests.h"
 #include "LogGymsSpatialFunctionalTest.h"
-#include "Modules/ModuleManager.h"
+#include "TestMapGeneration.h"
 
 DEFINE_LOG_CATEGORY(LogGymsSpatialFunctionalTest);
 
-IMPLEMENT_PRIMARY_GAME_MODULE( FDefaultGameModuleImpl, GDKTestGymsFunctionalTests, "GDKTestGymsFunctionalTests" );
+IMPLEMENT_GAME_MODULE(FGDKTestGymsFunctionalTestsModule, GDKTestGymsFunctionalTests)
+
+#define LOCTEXT_NAMESPACE "FGDKTestGymsFunctionalTestsModule"
+
+void FGDKTestGymsFunctionalTestsModule::StartupModule()
+{
+   FCoreDelegates::OnFEngineLoopInitComplete.AddLambda([]() {
+	   SpatialGDK::TestMapGeneration::GenerateTestMaps();
+   });
+}
+
+
+#undef LOCTEXT_NAMESPACE
```

Comment:
new lines at the end of the file please, just to get rid of this scary-looking icon in GitHub diffs

Reply:
(but of course, this is just a nit)

Reply:
Will add newlines for both files